stable/chartmuseum/templates/deployment.yaml

```diff
@@ -89,3 +98,20 @@ spec:
       {{- else }}
         emptyDir: {}
       {{- end -}}
+      {{ if .Values.env.open.GOOGLE_SERVICE_ACCOUNT }}
+      - name: {{ include "chartmuseum.fullname" . }}-gcp
+        secret:
+      {{ if .Values.env.open.GOOGLE_CREDENTIALS_JSON }}
+          secretName: {{ include "chartmuseum.fullname" . }}
```

Comment:
can we do "-gcp" here too? Does it matter?

Reply:
This `secretName` is the name used in the `secret.yaml` file. The `-gcp` is the name of the volume. We don't need to be dynamically generated, but it has to match the name used here:

```yaml
volumeMounts:
        - mountPath: /etc/secrets/google
          name: {{ include "chartmuseum.fullname" . }}-gcp
```